From 7710040ea723def69dde485d6f497fac87f215fa Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess at hadess.net>
Date: Sun, 11 Aug 2013 18:58:20 +0200
Subject: [PATCH] data: Fix syntax error in fprintd.pod

fprintd.pod around line 100: =over without closing =back
POD document had syntax errors at /usr/bin/pod2man line 69.
---
 data/fprintd.pod | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/fprintd.pod b/data/fprintd.pod
index 83c7afc..7085920 100644
--- data/fprintd.pod
+++ data/fprintd.pod
@@ -100,3 +100,6 @@ By default, fprintd stores the fingerprints in B</var/lib/fprint/>
 =over 8
 
 =item B<dbus-daemon>, B<gnome-about-me>
+
+=back
+
-- 
1.8.3.1

